packages/p/pocketbase.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "pocketbase",
+    "description": "PocketBase JavaScript SDK",
+    "author": "Gani Georgiev",

```

Comment:
This should be `authors`, and should be an array of objects, each with a `name` property and an optional `url`